pandas/core/arrays/categorical.py

```diff
@@ -2076,7 +2076,7 @@ def _reverse_indexer(self) -> Dict[Hashable, np.ndarray]:
         return result
 
     # reduction ops #
-    def _reduce(self, name, axis=0, **kwargs):
+    def _reduce(self, name: str, skipna: bool = True, **kwargs):
```

Comment:
Any API implications here? On master, specifying `axis=0` works

```python
In [10]: pd.Series(pd.Categorical([1, 2], ordered=True)).min(axis=0)
Out[10]: 1

```

Reply:
i don't think so, behaviour is the same

this PR:
```
>>> pd.__version__
'1.1.0.dev0+2121.g2f46f9eb9'
>>> pd.Series(pd.Categorical([1, 2], ordered=True)).min(axis=0)
1
>>> pd.Series(pd.Categorical([1, 2], ordered=True)).min(axis=1)
Traceback (most recent call last):
  File "C:\Users\simon\pandas\pandas\core\generic.py", line 368, in _get_axis_number
    return cls._AXIS_TO_AXIS_NUMBER[axis]
KeyError: 1

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\simon\pandas\pandas\core\generic.py", line 11456, in stat_func
    return self._reduce(
  File "C:\Users\simon\pandas\pandas\core\series.py", line 4223, in _reduce
    self._get_axis_number(axis)
  File "C:\Users\simon\pandas\pandas\core\generic.py", line 370, in _get_axis_number
    raise ValueError(f"No axis named {axis} for object type {cls.__name__}")
ValueError: No axis named 1 for object type Series
>>>
```

on master
```
>>> pd.__version__
'1.1.0.dev0+2120.g1b16a79c1'
>>> pd.Series(pd.Categorical([1, 2], ordered=True)).min(axis=0)
1
>>> pd.Series(pd.Categorical([1, 2], ordered=True)).min(axis=1)
Traceback (most recent call last):
  File "C:\Users\simon\pandas\pandas\core\generic.py", line 368, in _get_axis_number
    return cls._AXIS_TO_AXIS_NUMBER[axis]
KeyError: 1

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\simon\pandas\pandas\core\generic.py", line 11456, in stat_func
    return self._reduce(
  File "C:\Users\simon\pandas\pandas\core\series.py", line 4223, in _reduce
    self._get_axis_number(axis)
  File "C:\Users\simon\pandas\pandas\core\generic.py", line 370, in _get_axis_number
    raise ValueError(f"No axis named {axis} for object type {cls.__name__}")
ValueError: No axis named 1 for object type Series
>>>
```